provider/gce/google/raw.go

```diff
@@ -375,28 +376,36 @@ var doOpCall = func(call opDoer) (*compute.Operation, error) {
 // waitOperation waits for the provided operation to reach the "done"
 // status. It follows the given attempt strategy (e.g. wait time between
 // attempts) and may time out.
-//
-// TODO(katco): 2016-08-09: lp:1611427
-func (rc *rawConn) waitOperation(projectID string, op *compute.Operation, attempts utils.AttemptStrategy, f handleOperationErrors) error {
+func (rc *rawConn) waitOperation(projectID string, op *compute.Operation, retryStrategy retry.CallArgs, f handleOperationErrors) error {
 	// TODO(perrito666) 2016-05-02 lp:1558657
 	started := time.Now()
 	logger.Infof("GCE operation %q, waiting...", op.Name)
-	for a := attempts.Start(); a.Next(); {
+
+	retryStrategy.IsFatalError = func(err error) bool {
```

Comment:
This can be hoisted to the `longRetryStrategy` location.

Reply:
I suppose I could, but I'd rather not move the code too far away from where it's run. Especially since no repetition would be needed

Also for tradition's sake, to me the `Strategy` suffix is just sorta the general timings and retries, and less the specifics of the code that runs. Should probably have it's own struct somewhere